samples/angular-todo/src/app/features/journey/form/form.component.ts

```diff
@@ -161,7 +199,17 @@ export class FormComponent implements OnInit {
    */
   handleStep(step?: FRStep) {
     this.step = step;
-
+    const selectIdCallback = step.callbacks.find((cb) => cb.payload.type === 'SelectIdPCallback');
```

Comment:
Typo in the name, can we change this to `selectIdPCallback`?

Reply:
done